cargo-crates-io-deps-only.py

```diff
@@ -17,15 +17,23 @@ def manifests():
 
 re_git_or_path = re.compile('^\s*(git|path)\s*=.*$')
 
+has_printed_something = False
 for path in manifests():
 	manifest = open(os.path.join(path, CARGO_TOML), 'r')
 	with open(os.path.join(path, CARGO_TOML_NEW), 'w') as new_manifest:
+		print_something = False
 		for line in manifest:
 			if re_git_or_path.match(line):
+				print_something = True
 				continue
 			new_manifest.write(line)
+		if print_something:
+			print("Updating '%s'" % os.path.join(path, CARGO_TOML_NEW))
+			has_printed_something = True
 	manifest.close()
 	os.remove(os.path.join(path, CARGO_TOML))
```

Comment:
It'd be nice to remove `CARGO_TOML_NEW` and skip the renaming if there were no changes.